.github/workflows/test.yml

```
@@ -0,0 +1,24 @@
+name: iOS starter workflow
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main", "session/*" ]
+
+jobs:
+  build:
+    name: testWorkflow
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: UnitTest
+        run: xcodebuild test -project ios-training-bjung.xcodeproj/ -scheme ios-training-bjung -resultBundlePath TestResults -sdk iphonesimulator -destination "platform=iOS Simulator,name=iPhone 14"
```

[imo] xcbeautifyすると不要そうなログがでないのでおすすめ
```suggestion
        run: set -o pipefail && xcodebuild test -project ios-training-bjung.xcodeproj/ -scheme ios-training-bjung -resultBundlePath TestResults -sdk iphonesimulator -destination "platform=iOS Simulator,name=iPhone 14" | xcbeautify
```

---

xcresulttool 入れたら、確かに TimeZone の問題なのがわかりましたね。
まず xcbeautify を入れてプッシュしました。

---

@novr 
TimeZone 指定でテストが通りました。